src/lib/messenger.js

```diff
@@ -34,7 +34,8 @@ export default function messenger(bot) {
       } else if (inputParser.isAskingForBoegivar(text)) {
         handleBoegivar(bot, message.chat, user.firstName);
       } else if (inputParser.isAskingForTourettes(text)) {
-        handleTourettes(bot, message.chat, text);
+        const commandText = text.substr(11, text.length);
```

Comment:
Yeah, that's a great idea, I thought about it first using an array of supported commands and shit but in reality it would be way better to just use something like:

`
const text = stripText(text).text;

  if (hasCommand) {
    const command = stripText(text).command;
    ... //Send command and text to inputParser, else ignore
  }

function stripText(input) {
  var text = '';
  var command = '';
  var hasCommand = false;

  if (input.indexOf(' ') != -1 && input.startsWith('/')) {
    command = input.split(' ')[0];
    hasCommand = true;
    text = substr.input(command.length, input.length).trim();
  }
  else {
    text = input.trim();
  }
  
  return {
  text : text,
  command : command,
  hasCommand : hasCommand,
  };
}
`